AudioDeviceExample.xcodeproj/project.pbxproj

```diff
@@ -371,8 +371,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/twilio/twilio-voice-ios";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 6.2.0;
+				kind = upToNextMinorVersion;
```

Comment:
Does this mean, any patch version (eg. 6.13.4, 6.13.9 etc) will be auto updated but 6.14.0 will not?